src/clients/node-client.ts

```diff
@@ -1,12 +1,19 @@
 import { Client } from './client'
 import { Config, TokenSearchRules, TokenOptions } from '../types'
 import { Token } from '../token'
+import { PACKAGE_VERSION } from '../package-version'
 
 class MeiliSearch extends Client {
   tokens: Token
 
   constructor(config: Config) {
-    super(config)
+    super({
+      ...config,
+      clientAgents: [
+        ...(config.clientAgents ?? []),
+        `Meilisearch Node (v${PACKAGE_VERSION})`,
```

Comment:
Considering that this package might be used by any other runtime like Deno and Bun and more, does it make sense to call it Node? Maybe it would be best if we'd only ship one client, one that runs everywhere, and is a lot more treeshakeable, but that's part of another issue.